src/validation.js

```diff
@@ -182,11 +172,12 @@ export default class AuroFormValidation {
 
       this.getErrorMessage(elem);
 
-      elem.dispatchEvent(new CustomEvent(`${this.getCamelCaseName(elem)}-validated`, {
+      elem.dispatchEvent(new CustomEvent('auroFormElement-validated', {
         bubbles: true,
         composed: true,
         detail: {
-          validity: elem.validity
+          validity: elem.validity,
+          message: elem.errorMessage
```

Comment:
I wonder if we should send this hear or if we should also just query the values directly from the element (rather than evt.details). 

No change request at this time, just thinking out loud.

Reply:
Both ways would make sense since the elements values are getting set within the method, so we don't necessarily even need an event? It might be beneficial to a user though to have an explicit place where they can reliably say "the element was validated at this time".

Reply:
I think we do need the event. I'm just not sure if we should be having the components that use the library rely on the event.details for the validation result rather than just using the event as a trigger to go query the current validation state.